src/agent/dynamic-library/src/windows.rs

```diff
@@ -193,15 +196,19 @@ impl ImageLoaderSnapsGuard {
     }
 
     fn enable(&self) -> Result<(), ImageGlobalFlagsError> {
-        let mut value = self.gflags.get_value()?;
+        // Value may not yet exist.
+        let mut value = self.gflags.get_value().unwrap_or(0x0);
```

Comment:
Ideally `get_value()` would return an `Option`, but it doesn't, and the error is just a normal `io::Error`. If we really can't do registry I/O at all, we'll fail earlier (or in `set_value()`).